Content.Client/CriminalRecords/CriminalRecordsConsoleWindow.xaml.cs

```diff
@@ -167,19 +162,58 @@ public void UpdateState(CriminalRecordsConsoleState state)
         }
     }
 
-    private void PopulateRecordListing(Dictionary<uint, string> listing)
+    private void PopulateRecordListing(Dictionary<uint, string>? listing)
     {
-        _isPopulating = true;
+        if (listing == null)
+        {
+            RecordListing.Clear();
+        }
 
-        foreach (var (key, name) in listing)
+        var entries = listing!.ToList();
```

Comment:
Allowing null inputs then null suppressing is... interesting.

Going to checkout this pr but personally I don't like taking null input adding surprise changes to the method.